ejp-rd/.htaccess

```
@@ -0,0 +1,6 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^fairdatapoints/bbmri?(.*) https://ejprd.fair-dtls.surf-hosted.nl/bbmri-fdp/$1 [R=307,L]
+RewriteRule ^fairdatapoints/orphanet-catalog-fdp?(.*) https://ejprd.fair-dtls.surf-hosted.nl/orphanet-catalog-fdp/$1 [R=307,L]
+RewriteRule ^fairdatapoint-index?(.*) http://ejprd.fair-dtls.surf-hosted.nl:8070/$1 [R=307,L]
+RewriteRule ^triplestores/graphdb?(.*) http://ejprd.fair-dtls.surf-hosted.nl:7200/$1 [R=307,L]
```

The `?` in all these patterns is going to make the prior *letter* optional, which I assume is wrong.  Perhaps you meant to have an optional trailing `/` before the trailing text match?  If so, use something like this:

```suggestion
RewriteRule ^fairdatapoints/bbmri/?(.*) https://ejprd.fair-dtls.surf-hosted.nl/bbmri-fdp/$1 [R=307,L]
RewriteRule ^fairdatapoints/orphanet-catalog-fdp/?(.*) https://ejprd.fair-dtls.surf-hosted.nl/orphanet-catalog-fdp/$1 [R=307,L]
RewriteRule ^fairdatapoint-index/?(.*) http://ejprd.fair-dtls.surf-hosted.nl:8070/$1 [R=307,L]
RewriteRule ^triplestores/graphdb/?(.*) http://ejprd.fair-dtls.surf-hosted.nl:7200/$1 [R=307,L]
```

Note that this isn't fully catching all errors on the w3id.org side, as bbmriFOO would get through as bbmri-fdp/FOO. Rules could be adjusted to handle this, but it's easier to just ignore the garbage-in, garbage-out situation unless it's a problem for correct inputs.

---

@davidlehn Thank you for the review. Yes, I meant to have trailing `/` in the URLs. I missed it.

---

About `bbmri-fdp/FOO` case it is a bit hard to write rules against it. The FDP software used in this project generates thousands of RDF records which need to have PIDs. I will investigate if the suffix of these records can be fixed.